.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,30 @@
 <!--Please use the template provided below when reporting bugs:-->
-## Link to bug demonstration repository.
+## Link to bug demonstration repository
 <!--
 A demo repository will show the combination of module versions being used,
 command-lines and configurations.  NYC is very complex, it is difficult or
 even impossible to triage most bugs without this information.
 
 If you cannot share a link to your repository (maybe it's private) then you must
-create a basic `nyc-bug-demo` repository and link to that please.
+create a basic `nyc-bug-demo` repository and link to that please [mandatory].
 -->
 
 ## Expected Behavior
 
 ## Observed Behavior
 
-## Forensic Information
+### Troubleshooting steps
+- [ ] still occurrring when I put `cache: false` in my nyc config
 
-**Operating System:** the operating system you observed the issue on.
-**Environment Information:** information about your project's environment, see instructions below:
-
-1. run the following script:
+## Environment Information
+<!--
+[mandatory] run the following script: npx envinfo --preset nyc
```

Comment:
because envinfo is included in a LOT of projects now, you may want to specify a version or `npx envinfo@latest`, or you may get an old version before your preset was added.